.dockerignore

```diff
@@ -0,0 +1,10 @@
+.git/
+.dockerignore
+**/.gitignore
+**/c.out
```

Comment:
What is generating these?

Reply:
there is `make coverage` target in the `Makefile` that generates `c.out` file which is input to `go tool` to generate `coverage.html`